src/components/info-window.tsx

```diff
@@ -19,38 +25,76 @@ export const InfoWindow = (props: PropsWithChildren<InfoWindowProps>) => {
   const {children, anchor, onCloseClick, ...infoWindowOptions} = props;
   const map = useContext(GoogleMapsContext)?.map;
 
+  const infoWindow = useRef<google.maps.InfoWindow | null>(null);
```

Comment:
Since it's not the infowindow itself here, can we rename this to `infoWindowRef`?

Reply:
Sure :)